textgrad/engine/bedrock.py

```diff
@@ -0,0 +1,157 @@
+try:
+    import boto3
+    from botocore.config import Config
+
+except ImportError:
+    raise ImportError("If you'd like to use Amazon Bedrock models, please install the boto3 package by running `pip install boto3`")
```

Comment:
can we add the related environment variables that would be needed here? i.e. AWS_ACCESS_KEY_ID etc,

Reply:
Lines 63/68 handles the variables needed. But if you prefer I can move them also here